webapp/tests/elster_client/mock_erica.py

```diff
@@ -291,6 +295,21 @@ def get_address_data(input_body, show_response: bool):
             else:
                 return get_json_response('get_address_no_resp')
 
+    @staticmethod
+    def is_valid_tax_number(state_abbreviation, tax_number):
+        if err_response := MockErica.errors_from_error_flags(True):
+            return err_response
+
+        _VALID_TAX_NUMBERS = ['19811310010']
+        _VALID_STATE_ABBREVIATIONS = ['BW', 'BY', 'BE', 'BB', 'HB', 'HH', 'HE', 'MV', 'ND', 'NW', 'RP', 'SL', 'SN', 'ST', 'SH', 'TH']
+
+        if MockErica.tax_number_is_invalid \
```

Comment:
I think, instead of handling the `MockErica.tax_number_is_invalid` check here, we should also include that into `MockErica.errors_from_error_flags`.

Reply:
Yes, we could do that and I thought about. However, this is not an error that Erica would throw but just a flag for us to indicate that the MockErica should return False